episode2.py

```diff
@@ -112,15 +113,15 @@ def update(self, dt):
             item.velocity += g * dt
 
         if len(self.drugs) < 15 and random.randint(0, 100) < 10:
-            drug = arcade.Sprite("img/coin.png", SPRITE_SCALING_COIN)
+            drug = arcade.Sprite("img/Broccoli2.png", SPRITE_SCALING_COIN)
```

Comment:
Well, in this case, this is not drug anymore. It is better to remove code occurences in order to not misguide the readers.